tests/ui/llm_studio_page.py

```diff
@@ -0,0 +1,242 @@
+from hac_playwright.pages.base import BasePage
+from playwright.sync_api import expect
+
+
+class LLMStudioPage(BasePage):
+    # Constants for selectors
+    DATASET_IMPORT_SOURCE_SELECTOR = "dataset/import/source"
+    CONTINUE_BUTTON_SELECTOR = "button[name='Continue']"
+    DATASET_LIST_DELETE_SELECTOR = "dataset/list/delete"
+    DATASET_DELETE_DIALOG_SELECTOR = "dataset/delete/dialog"
+    DATASET_DELETE_SELECTOR = "dataset/delete"
+    EXPERIMENT_START_RUN_SELECTOR = "experiment/start/run"
+    EXPERIMENT_NAME_SELECTOR = "experiment/start/cfg/experiment_name"
+    EXPERIMENT_LIST_DELETE_SELECTOR = "experiment/list/delete"
+    EXPERIMENT_DELETE_DIALOG_SELECTOR = "experiment/delete/dialog"
+    EXPERIMENT_DELETE_SELECTOR = "experiment/delete"
+    EXPERIMENT_STATUS_SELECTOR = "[data-automation-key='status']"
+    EXPERIMENT_INDEX_SELECTOR = "[data-automation-key='name']"
+    FILESYSTEM_PATH = "/home/llmstudio/mount/data/user/"
```

Comment:
This filepath won't work for a locally running instance.

Reply:
1. this should be passed from the outside of the tests and be part of test configuration
2. the file path doesn't belong to the Page Object, which is the abstraction level for the UI actions, but rather to the test itself